server/controllers/login.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require("bcrypt");
+const Boom = require("@hapi/boom");
+const { validateSchema, signToken } = require("../utils");
+const { getUser } = require("../database");
+
+const login = async (req, res, next) => {
+  try {
+    const { email, password } = req.body;
+    let userID;
+    try {
+      await validateSchema.validateAsync({ email, password });

```

Comment:
set { abortEarly: false } in option to returns all the errors found.

Reply:
done